src/main/java/com/hazelcast/aws/exception/AwsConnectionException.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
```

Comment:
that seems an old header definition. We might need to update whole codebase though.

Reply:
ok, added as #61